tests/core/mountpoint/test_winfsp_tests.py

```diff
@@ -3,14 +3,30 @@
 import os
 import pytest
 
+
 if os.name != "nt":
     pytest.skip("Windows only", allow_module_level=True)
 
-from winfspy.tests.winfsp_tests import test_winfsp_tests as winfsp_tests
+try:
+    from winfspy.tests.winfsp_tests import test_winfsp_tests as winfsp_tests
+
+except RuntimeError as exc:
+    if "The `winfsp-tests-x86` executable cannot be found" in str(exc):
+        supported_test_cases_marker = pytest.mark.skip(
+            reason="winfsp-tests-x86 executable not installed"
+        )
+
+        @pytest.fixture
+        def file_system_tempdir():
+            pass
 
-# Constants
+    else:
+        raise
+
+else:
+    supported_test_cases_marker = pytest.mark.parametrize("test_case", winfsp_tests.TEST_CASES)
+    file_system_tempdir = winfsp_tests.file_system_tempdir
```

Comment:
Why not simply use `pytest.skip("...", allow_module_level=True) ` like here?

https://github.com/Scille/parsec-cloud/blob/763e050cdbbc5cf9ec49c5e9b307b92939b0221e/tests/core/mountpoint/test_winfsp_tests.py#L6-L7

Reply:
Nice ! I didn't know about this ;-)